pmd-cataloger/src/main/java/sfdc/sfdx/scanner/pmd/Pmd7CompatibilityChecker.java

```diff
@@ -13,7 +13,10 @@
  */
 public class Pmd7CompatibilityChecker {
     private static final String BAD_PROP_TEMPLATE = "%s tag requires property '%s' to be %s";
-    private static final String EXPECTED_XPATH_CLASS = "net.sourceforge.pmd.lang.rule.XPathRule";
+    private static final String BAD_PROP_TEMPLATE2 = "%s tag requires property '%s' to be one of the following: %s";
```

Comment:
If there are multiple templates, we should probably give them better names than `1` and `2`. Can we name them something like `single_option` and `multi-option`?

Reply:
Sure thing. Fixed, rebased, and force pushed.